### PR TITLE
keep volume aligned to 32 when using systemmenu

### PR DESCRIPTION
**Pull request by jwunderl**
(also remove redundant mins/maxes as that's handled in setBrightness)

### CODE REVIEW

**Comment by mmoskal** [comment]
Maybe it would be better to have non-linear jumps for volume? our hearing response is logarithmic; though I guess people will wonder why are they getting random values...

**Comment by jwunderl** [comment]
Maybe non linear jumps and just show a progress bar to indicate 'relative loudness'?

**Comment by mmoskal** [comment]
I think you also want the number in case they want to call `music.setVolume()`. I think that one should stay with amplitude, since this agrees with envelopes etc.

**Comment by mmoskal** [comment]
though, maybe there should be no `music.setVolume()`; we should just persist user setting...

**Comment by jwunderl** [comment]
A block for setting volume does seem like a pretty easy way for someone to make a jumpscare game now that i think about it (e.g. leave it alone and then halfway through the game max out volume in a game update and play the loudest screech you can).

File: libs/game/systemmenu.ts [hunk 1]
```diff
@@ -269,16 +269,18 @@ namespace scene.systemMenu {
 
     function volumeDown() {
         const v = music.volume();
-        music.setVolume(v - 32);
+        const newVolume = ((v - 1) >> 5) << 5;
```
Comment (pelikhan) [inline libs/game/systemmenu.ts]: when v = 0, newVolume is -32. is that expected?

Comment (pelikhan) [reply libs/game/systemmenu.ts]: Can we do less magic for this?

Comment (jwunderl) [reply libs/game/systemmenu.ts]: Yeah, I was letting setVolume ignore it. I'll rewrite it to be less bit shifty

**Pull request merged**